internal/eventstore/handler/handler_projection.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/caos/zitadel/internal/eventstore"
 )
 
+const systemID = "system"
```

Comment:
We will have to change the spooling part to single instances. This will take some time and we don't want to delay V2 for that. But as of now the signing keys don't have the exact same mechanism and already need to lock and check per instance. Therefore we added a new column `instance_id` on the locks table which the spooler will have to provide a value for: `system`